### DISCUSSION TITLE
Livewire 3 and Sortable Issue: window.livewire is undefined

### DISCUSSION THREAD

**Discussion by reasecret** [livewire/livewire#6649] [1. Help] [answered]
I'm trying to use sortable plugin with Livewire 3 but getting error on console:
`Uncaught Livewire Sortable Plugin: window.livewire is undefined. Make sure @livewireScripts is placed above this script include`

Livewire 3 not required @livewirescripts directive but I've added this directive to my layout blade but still getting this error. Any ideas?

**Comment by jdesousa1** [comment]
Hi @reasecret 

Did you tried with $wire instead of window.livewire?

**Reply by reasecret** [reply]
@jdesousa1 I'm not using window.livewire but cdn file is. Using plugin via cdn.

**Reply by satoved** [reply, marked as answer]
livewire-sortable isn't supporting Livewire 3 yet, see pull request https://github.com/livewire/sortable/pull/67

**Reply by reasecret** [reply]
Thanks. Waiting for new release...

**Comment by yabdab** [comment]
So has this been fixed? Getting the following error still in 2024 LW 3 and Laravel 11 

```
index.js:3 Uncaught Livewire Sortable Plugin: window.livewire is undefined. Make sure @livewireScripts is placed above this script include
```

**Reply by yabdab** [reply]
nevermind, just had to use newest cdn URL  
`<script src="https://cdn.jsdelivr.net/gh/livewire/sortable@v1.x.x/dist/livewire-sortable.js"></script>`
